### PR TITLE
ADS-630: Replace border styles with tokens

### PR DESCRIPTION
**Pull request by cblva**

### CODE REVIEW

File: src/styles/components/_input-radio-button-group.scss [hunk 1]
```diff
@@ -122,11 +122,11 @@
           .bsds-input-radio-input {
             &:checked {
               + .bsds-input-radio-label {
-                border-color: $feedback-error-40;
+                border-color: $border-color-error;
 
                 &:hover,
                 &:active {
-                  border-color: $action-primary-20;
+                  border-color: $border-color-error;
```
Comment (kaseyllabelle) [inline src/styles/components/_input-radio-button-group.scss:129]: looks like this one should be $action-primary-20

File: src/styles/components/_input-text.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
 .bsds-input-text-input {
   width: 100%;
   margin-top: space(md);
-  border: space(xxs) solid $neutral-40;
+  border: $border-width-xxs solid $neutral-40;
```
Comment (kaseyllabelle) [inline src/styles/components/_input-text.scss:8]: we could ask catie if she wants to swap this to $border-color-assertive. text and select have $neutral-40, but checkbox and radio are $neutral-00

File: src/styles/components/_content-card.scss [hunk 3]
```diff
@@ -16,21 +16,21 @@
...
     padding: space(3x);
-    background-color: $neutral-100;
+    background: $neutral-100;
   }
 
   &-border-ghost {
     @extend .bsds-card-ghost;
     @extend .bsds-card-border-light;
-    background-color: rgba($neutral-100, 0.2);
+    background: rgba($neutral-100, 0.2);
```
Comment (APJohns) [inline src/styles/components/_content-card.scss:33]: Should this be `$border-color-ghost`?

File: src/styles/components/docs/_demo.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 .docs-preview {
   margin-top: space(3x);
-  border: 1px solid $neutral-00; // one-off to match markdown's background color
+  border: $border-width-xxs solid $border-color-assertive; // one-off to match markdown's background color
```
Comment (kaseyllabelle) [inline src/styles/components/docs/_demo.scss:3]: we can prob remove this comment now

File: src/styles/components/_input-radio-button-group.scss [hunk 5]
```diff
@@ -29,9 +29,9 @@
       @include font-family-base-regular;
       outline: space(xxs) solid $neutral-40;
       border-radius: space(3x);
-      border: space(xs) solid transparent;
+      border: $border-width-xxs solid transparent;
```
Comment (APJohns) [inline src/styles/components/_input-radio-button-group.scss:32]: Was this change of xs -> xxs intentional?

File: src/styles/components/_content-card.scss (hunk 3 above)
Comment (kaseyllabelle) [reply src/styles/components/_content-card.scss:33]: this is for the translucent background, so we can leave it as neutral-100

**Review by APJohns — APPROVED** [review]

**Pull request merged**